### PR TITLE
Add helperText prop to ArrayInput

### LINKED ISSUE #4227: ArrayInput does not support helperText prop
**What you were expecting:**
As ArrayInput should accept the common input props, helperText should be working, too.

**What happened instead:**
HelperText prop is not handled in the input, therefore does not appear.

**Related code:**
I'd love to create a pull request, however, I am not sure how the related files are compiled. Making the prop work would only need a few lines of code into ArrayInput.tsx:

```
import {  InputLabel, FormControl, FormHelperText } from '@material-ui/core';
...
<InputLabel ... />
<FormHelperText>{helperText}</FormHelperText>
```
Besides this, adding helperText const to the props and the proptypes.

**Environment**

* React-admin version: 3.1.1

### PR DESCRIPTION
**Pull request by netdown**
Fixes #4227, follow-up of #4262. Typings are fixed in the latest react-final-form-arrays release, therefore tests should be fine now.

### CODE REVIEW

File: packages/ra-ui-materialui/package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
         "react": "^16.9.0",
         "react-dom": "^16.9.0",
         "react-final-form": "^6.5.0",
-        "react-final-form-arrays": "^3.1.1",
+        "react-final-form-arrays": "^3.1.2",
```
Comment (Luwangel) [inline packages/ra-ui-materialui/package.json:45]: You should run `make install` (which runs `yarn install`) in order to upgrade the `yarn.lock` file. It's the only way to really upgrade a dependency.

See [Yarn documentation](https://yarnpkg.com)

**Comment by fzaninotto** [comment]
Not sure why this got closed. I usually leave a comment when closing issues, so it must be a consequence of deleting the `next` branch during a minor release. Sorry for the bad signal.

Reopening. Please read the CI report and review and make the requested changes to have it merged.

**Comment by fzaninotto** [comment]
This was closed automatically when I merged next to master - sorry for that. Reopening.

**Comment by fzaninotto** [comment]
This was closed automatically when I merged next to master - sorry for that. Reopening.

**Comment by WiXSL** [comment]
This can be closed

**Comment by djhi** [comment]
Superseded by #6062

**Pull request closed without merging**